packages/server/graphql/mutations/helpers/makePokerTemplateScales.ts

```diff
@@ -0,0 +1,50 @@
+import TemplateScale from '../../../database/types/TemplateScale'
```

Comment:
if scales can be re-used across templates on the same team we can get rid of this